sdk/storage/Azure.Storage.Files.Shares/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/ccae65b66ac23a9fd891ba86f45abc4372a9c688/specification/storage/data-plane/Microsoft.FileStorage/preview/2023-01-03/file.json
+    - C:\azure-rest-api-specs\specification\storage\data-plane\Microsoft.FileStorage\preview\2024-02-04\file.json
```

Comment:
TODO fix this.

Reply:
I have fixed this.  @tasherif-msft, note that the URL to the swagger file has to contain a commit hash.  The way to get this URL is to navigate to your fork, branch, and the file in question, get the raw view, and copy the URL.